### PR TITLE
Rename parameter scale to scales

### PR DESCRIPTION
**Pull request by skoudoro**
Quick PR to rename the parameter `scale` to `scales` before it becomes a habit.

### CODE REVIEW

File: docs/examples/viz_advanced.py [hunk 1]
```diff
@@ -264,7 +264,7 @@ def win_callback(obj, _event):
 # Finally, please set the following variable to ``True`` to interact with the
 # datasets in 3D.
 
-interactive = False
+interactive = True
```
Comment (Garyfallidis) [inline docs/examples/viz_advanced.py:267]: Should this be True?

Comment (skoudoro) [reply docs/examples/viz_advanced.py:267]: good catch! updated

**Comment by guaje** [comment]
The primitive module still has some scale params in singular: https://github.com/fury-gl/fury/blob/master/fury/primitive.py#L50

**Comment by skoudoro** [comment]
ok,  I will update that, thanks @guaje!

**Comment by guaje** [comment]
> ok, I will update that, thanks @guaje!

Also the superquadric function: https://github.com/fury-gl/fury/blob/master/fury/actor.py#L1864

**Comment by guaje** [comment]
> > ok, I will update that, thanks @guaje!
> 
> Also the superquadric function: https://github.com/fury-gl/fury/blob/master/fury/actor.py#L1864

**Comment by skoudoro** [comment]
ok, all fixed @guaje. thank you for the review. Let me know if you see something else concerning `scales`

**Comment by guaje** [comment]
> ok, all fixed @guaje. thank you for the review. Let me know if you see something else concerning `scales`

Sure, I'm checking again all the files. So far, I've found this:

- https://github.com/fury-gl/fury/pull/289/files#diff-8da821ba4630b32206fafffa55273b6dR1332

**Comment by guaje** [comment]
I can't find any missing usage of the word "scale". Merging.

**Pull request merged**